src/components/KInput/KInput.vue

```diff
@@ -216,6 +218,8 @@ const helpText = computed((): string => {
   return props.help
 })
 
+const helpTextId = helpText.value ? uuidv4() : ''
```

Comment:
```suggestion
const helpTextId = uuidv4()
```

let's just have this static const have a value at all times (since it's initialized on component setup)